static/css/components/observationStats.less

```diff
@@ -1,25 +1,67 @@
+@import (reference) "../less/colors.less";
+
+.review-component {
+  padding-bottom: 1em
+}
+
 .observation-title {
   margin-top: 5px;
-  margin-bottom: 0;
+  margin-bottom: 1em;
 }
 
-.observation-header {
-  font-size: 14px;
-  margin: 0 0 1em;
+.reviews {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  margin-bottom: .5em;
```

Comment:
Not a blocker; I would recommend using px; ems or _very_ confusing to deal with. I haven't looked too much into it, but anyone can understand and use pixels, whereas `em`s are very hard to reason about :P